src/hub/hub.ts

```diff
@@ -70,6 +70,10 @@ export default function hub(): Hub {
               tempRef.parentNode.removeChild(tempRef);
               break;
             }
+            case "navigation": {
+              document.getElementById("config").src = data.url;
```

Comment:
A bit more complicated - but you could have passed this into the iframe instead of replacing the url. Making the entire experience smoother.